src/migration/1687349045215-DisallowHandle.ts

```diff
@@ -0,0 +1,5518 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DisallowHandle1687349045215 implements MigrationInterface {
+  name = 'DisallowHandle1687349045215';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "disallow_handle" ("value" character varying NOT NULL, CONSTRAINT "PK_8c84e48365905231826700dd8b4" PRIMARY KEY ("value"))`,
+    );
+    await queryRunner.query(
+      `CREATE UNIQUE INDEX "IDX_8c84e48365905231826700dd8b" ON "disallow_handle" ("value") `,
+    );
+    await queryRunner.query(
```

Comment:
Can we turn it into a script perhaps? This will make the migration relatively heavy and we constantly run it for tests and so on

Reply:
Sure was my first thought, but then thought we might want them locally everything as we do with keywords now.
Will convert to script for now 🙏